plugin-server/src/main/ingestion-queues/events-ingestion-consumer.ts

```diff
@@ -0,0 +1,63 @@
+import { Message } from 'node-rdkafka'
+
+import { buildStringMatcher } from '../../config/config'
+import { prefix as KAFKA_PREFIX, suffix as KAFKA_SUFFIX } from '../../config/kafka-topics'
+import { Hub } from '../../types'
+import { status } from '../../utils/status'
+import { eachBatchParallelIngestion, IngestionOverflowMode } from './batch-processing/each-batch-ingestion'
+import { IngestionConsumer } from './kafka-queue'
+
+export type PipelineType = {
+    topic: string
+    consumer_group: string
+}
+
+export const PIPELINES: { [key: string]: PipelineType } = {
```

Comment:
Can we use the constants in `kafka-topics.ts` here?

Reply:
the tricky thing is I don't want the person who creates the pipeline to need to touch multiple files